tests/bdd/scale/test_NAS_T1357.py

```diff
@@ -116,4 +116,4 @@ def confirm_image_is_deleted(driver):
     """confirm image is deleted."""
     assert wait_on_element(driver, 10, '//*[contains(.,"Please wait")]')
     assert wait_on_element_disappear(driver, 30, '//*[contains(.,"Please wait")]')
-    assert wait_on_element_disappear(driver, 30, '//tr[contains(.,"machinaris")]') is False 
```

Comment:
@ericbsd that line has been replaced with the is_element_present line which proved to be more stable on my testing.  The page sometimes entirely reloads which means there isnt an element to 'disappear' because on page refresh it doesnt exist at all.